tests/test_did_meta_plugins.py

```diff
@@ -284,6 +285,107 @@ def test_list_did_meta(self, mock_scope, root_account, mongo_meta):
         # assert [{'scope': (tmp_scope), 'name': tmp_dsn4}] == results
         assert [tmp_dsn4] == results
 
+@pytest.fixture()
```

Comment:
```suggestion
@pytest.fixture
```

for consistency with the other fixtures without arguments